### ISSUE TITLE
Images on Docker Hub should be version-tagged

### ISSUE THREAD

**Issue by Ravenstine** [jhpyle/docassemble#126]
Images built on hub.docker.com are only tagged as latest.

https://hub.docker.com/r/jhpyle/docassemble

These pre-built images would be far more useful if they were tagged by application version, this way deployments and custom images that use them as a base can be reproducible.  It should be possible for the version of the application to be locked wherever it's used.

**Comment by jhpyle** [comment]
I updated the documentation to explain how to obtain Docker images for earlier versions.  See https://docassemble.org/docs/docker.html#downgrading

I experimented to see if I could reduce the size of the Docker image so that I could publish tagged images on Docker Hub.  However, after moving extra package installation from the Docker build process to the Docker start process, that only reduced the image size from 6GB to 4GB.

It is not feasible to publish Docker images for every version, given the size of the Docker image.  If I had been publishing tagged images, I would now be using up 1.5 terabytes of space on Docker Hub, which is not reasonable.
